style.css

```diff
@@ -16,22 +17,61 @@ main {
 	justify-content: flex-start;
 	gap: 50px;
 	padding: 15px;
+	padding-left: 45px;
 	background-color: #9a7b5c;
+	font-family: 'Roboto Flex', sans-serif;
+	font-size: 20px;
+	letter-spacing: 3px;
+	
 }
 
 .chloe-smart {
 	  position: absolute;
```

Comment:
when scaled down to 400px, this doesn't work (phone size) consider changing the formatting on the @media quiery later?